rfcs/text/0017-remove-log-original.md

```diff
@@ -1,7 +1,7 @@
 # 0017: Remove log.original
 
-- Stage: **2 (candidate)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
-- Date: **2021-04-28** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+- Stage: **3 (finished)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **2021-06-18** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
```

Comment:
Update when you are ready to merge 🚢

Reply:
thx! done, just waiting for last run thru ci...